packages/wrangler/src/__tests__/r2/r2.test.ts

```diff
@@ -0,0 +1,119 @@
+import { endEventLoop } from "../helpers/end-event-loop";
+import { mockConsoleMethods } from "../helpers/mock-console";
+import { runInTempDir } from "../helpers/run-in-tmp";
+import { runWrangler } from "../helpers/run-wrangler";
+
+describe("r2", () => {
+	const std = mockConsoleMethods();
+	runInTempDir();
+	it("should show help when no argument is passed", async () => {
+		await runWrangler("r2");
+		await endEventLoop();
+		expect(std.out).toMatchInlineSnapshot(`
+		"wrangler r2
+
+		📦 Interact with an R2 store
+
+		Commands:
```

Comment:
please make sure you rebase your branch on top of latest `main`. We've had some changes in there that would cause these tests to fail